weedcoco/importers/masks2.py

```diff
@@ -0,0 +1,125 @@
+from PIL import Image, ImageDraw
+import numpy as np
+from skimage import measure
+from shapely.geometry import Polygon, MultiPolygon
+
+
+def create_sub_masks(mask_image):
```

Comment:
I suspect you're overengineering this.
You should just convert the image bitmap into an array:
```py
im = PIL.Image.open(...)
arr = np.asarray(im)
```
arr is of shape (height, width, depth), where depth=3 for the cobbity masks.
You can see that there are three distinct colours with:
```py
distinct_colours = np.unique(np.reshape(arr, (-1, 3)), axis=0)
```
You can extract out the first of those colours (black) with:
```py
mask = (arr == distinct_colours[0]).all(axis=-1)
```

Thus:
![image](https://user-images.githubusercontent.com/78827/95952283-8ca2a880-0e43-11eb-8c9c-5b9d08d399d7.png)